hadoop-hdds/common/src/main/java/org/apache/hadoop/utils/db/TypedTable.java

```diff
@@ -106,6 +142,40 @@ public void close() throws Exception {
 
   }
 
+  @Override
+  public void addCacheEntry(CacheKey<KEY> cacheKey,
+      CacheValue<VALUE> cacheValue) {
+    // This will override the entry if there is already entry for this key.
+    cache.put(cacheKey, cacheValue);
+  }
+
+
+  @Override
+  public void cleanupCache(long epoch) {
```

Comment:
We can remove this function I think. Caller can do `table.getCache().cleanup()`.

Reply:
This is added because, in actual usage of classes, we get Table reference, and they can simply call table.cleanupCache(). Added getCache() only for testing, so declared it as not a public method and annotated with VisibleForTesting. This will be used after double-buffer integration completed.

Let me know if you still want to be removed?

Reply:
We can revisit this later.